honeybee_vtk/config.py

```diff
@@ -229,3 +250,38 @@ class Config(BaseModel):
         description='List of data-config objects that define the data to be loaded on'
         ' the model.'
     )
+
+
+class DateTimeConfig(BaseModel):
+    """A DateTime object."""
+    month: conint(ge=1, le=12) = Field(
+        description='Month of the year. Accepted values are between 1 and 12 inclusive.'
+    )
+    day: conint(ge=1, le=31) = Field(
+        description='Month of the year. Accepted values are between 1 and 31 inclusive.'
+    )
+    hour: conint(ge=0, le=23) = Field(
+        description='Hour of the day. Accepted values are between 0 and 23 inclusive.'
+    )
```

Comment:
How do we handle timesteps smaller than one?

Reply:
So the way thresholding will work is, all the timesteps between two DateTimes will be caught. Say for example, the time step is one hour and DateTime is (6,21,8) to (6.21,9) then 1 timestep will be caught. If the time steps if 0.5, two timesteps will be caught.